itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCrudCompactorOnTez.java

```diff
@@ -146,7 +146,7 @@ private void testRebalanceCompactionWithParallelDeleteAsSecond(boolean optimisti
       }
       aborted = true;
       Assert.assertEquals(e.getCause().getClass(), LockException.class);
-      Assert.assertEquals(e.getCauseMessage(), "Transaction manager has aborted the transaction txnid:21.  Reason: Aborting [txnid:21,24] due to a write conflict on default/rebalance_test committed by [txnid:20,24] d/u");
+      Assert.assertEquals( "Transaction manager has aborted the transaction txnid:21.  Reason: Aborting [txnid:21,24] due to a write conflict on default/rebalance_test committed by [txnid:20,24] d/u", e.getCauseMessage());
```

Comment:
I like that you changed the order of the parameters as the first parameter is the expected. However, now it bugs my mind that the other assertion in the exact previous line still has the wrong order of parameters. Could you please change that as well?

Reply:
I can understand your point, but the build environment is really unstable, and I do not want to risk a failed build because of this change.